docs/source/development-guide/toc.rst

```diff
@@ -17,6 +17,7 @@ If you have questions or would like to help please open an issue on GitHub_.
    make/toc
    logging
    documentation
+   workers/toc
```

Comment:
There is no docs for the workers, it does not make sense to include this TOC entry. This might be an old change too, but it needs to be removed.